sdk/lib/opentelemetry/sdk/trace/export/in_memory_span_exporter.rb

```diff
@@ -20,20 +20,28 @@ module Export
         #   end
         #
         #   def test_finished_spans
+        #     @exporter.recording = true
         #     @tracer_provider.tracer.in_span("span") {}
         #
         #     spans = @exporter.finished_spans
         #     spans.wont_be_nil
         #     spans.size.must_equal(1)
         #     spans[0].name.must_equal("span")
+        #
+        #     @exporter.recording = false
         #   end
         class InMemorySpanExporter
+          # Controls whether or not the exporter will record spans, or discard them.
+          # @return [Boolean] when true, the exporter is recording. By default, this is false.
+          attr_accessor :recording
+
           # Returns a new instance of the {InMemorySpanExporter}.
           #
           # @return a new instance of the {InMemorySpanExporter}.
           def initialize
```

Comment:
Could we do an optional param for the initializer?  That way we don't have to set the recording value everywhere?  
```suggestion
          def initialize(recording: true)
```

I think defaulting to false might be a little confusing too if you're just playing around with the lib and it's not obvious why you have no spans.

Reply:
We could, yes. My thought was that defaulting to `false` was nice for users writing tests against the SDK in their own apps; but I suppose we could make that nicer via an example docs of how to flip recording on and off. 😄

Reply:
I've made that change, and force-pushed the branch so it's a nice commit history. 😄